Enemies of Nanotrasen: A Quick Overview (a new in-game manual for the crew that briefly covers antags in an IC way)

DeityLink opened the pull request:
I had been talking for years about something like that. Here's a PDF for proper viewing, feel free to offer suggestions on information to add or wording to modify.

[eighthrevision.pdf](https://github.com/vgstation-coders/vgstation13/files/12268340/eighthrevision.pdf)

![image](https://github.com/vgstation-coders/vgstation13/assets/7573912/946a7253-6d73-4593-9f30-34fc25ba60f5)

This is basically a companion book to Space Law that goes into detail over what that first book means by "Enemies of the corporation", but also provides the crew (and especially new players) some insight on how to handle some types of antags, especially ones that either require unconventional ways to fight them (such as grues, pulse demons), that can only be permanently killed a certain way (like vampires, changelings), or that have some features that the crew ought to know (for exemple how to deconvert revs/cultists, removing wizard robes to neuter them, etc)

:cl:
* rscadd: A new manual has been added that can notably be found in Security and at the Library, that covers in an IC way some basic information on every antags and more importantly how to deal with them.

west3436 commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<body style="font-size: 18px;color:#F7D61A;background-color:#221111;">
+
+				<h1><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "nano-logo"))]' style='position: relative; top: 6;'>Enemies of Nanotrasen: A Quick Overview<img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "nano-logo"))]' style='position: relative; top: 6;'></h1>
+                <h2>Forword</h2>
```
Foreword

DeityLink replied:
```suggestion
                <h2>Foreword</h2>
```

Eneocho commented:
A lot of sentences are really weirdly phrased, but this is a good change.
Also, it's "Battle Royale", not "Battle Royal", on the challengers' tab.

SonixApache commented:
good addition

don't like the challenger into nuke ops lore

SonixApache commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p>The Syndicate trains its traitors to become nuclear operatives through a Battle Royal in which some of its agents are targeting each other. On the bright side, those challengers are here to kill each other. On the downside, they don't care about how much collateral damage they cause, AND the victor may graduate to become a fully fledged nuclear operative one day. We do not want that. Chall ...
+
+				<h3>Elite Strike Team</h3>
+				<p>The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations are promoted to wear these black rigsuits, crafted from what we assume to be retro-engineered technology from our own special ops teams. If they come, the station is almost certainly doomed, so do your utmost to take them with you, so that they won't live to attack a ...
```
do not mention the squid - ever
```suggestion
				<p>The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations. Sightings are rare, but they generally seem to field upgraded black rigsuits and extremely lethal ballistic weaponry. Their arrival is almost certain to spell doom for any station in a sector; do what you can to stop or delay them. 
```

DeityLink commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p>A highly trained team of Syndicate agents, usually equipped with red rigsuits and 12mm SMGs. They are not only highly dangerous to fight but even to kill, as their brains are rigged with explosive implants that will try and take their aggressor's life with theirs, keep your distance as you fight them. Their goal is to acquire the station captain's nuclear authentication disk, so they can p ...
+
+				<h3>Challengers</h3>
+				<p>The Syndicate trains its traitors to become nuclear operatives through a Battle Royal in which some of its agents are targeting each other. On the bright side, those challengers are here to kill each other. On the downside, they don't care about how much collateral damage they cause, AND the victor may graduate to become a fully fledged nuclear operative one day. We do not want that. Chall ...
```
```suggestion
				<p>The Syndicate trains its novice agents through a Battle Royale where they have to eliminate each others aboard one of our stations. On the bright side, those challengers are here to kill each other. On the downside, they don't care about how much collateral damage they cause, AND the victor may graduate to become a fully fledged Syndicate agent, enabling them to cause even more trouble. We do not want that. Challengers often grimly display the corpse of their target on Newscasters, use this opportunity to try and figure out the identity of the victor so you may deal with them.
```

DeityLink commented:
@SonixApache I fixed it, forgot I made it as a trial for new agents, not as a graduation into nuke ops. #28207

ShiftyRail commented:
I am neutral on the idea but I am not sure if the tone/style is the best.

ShiftyRail commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p>A highly trained team of Syndicate agents, usually equipped with red rigsuits and 12mm SMGs. They are not only highly dangerous to fight but even to kill, as their brains are rigged with explosive implants that will try and take their aggressor's life with theirs, keep your distance as you fight them. Their goal is to acquire the station captain's nuclear authentication disk, so they can p ...
+
+				<h3>Challengers</h3>
+				<p>The Syndicate trains its novice agents through a Battle Royale where they have to eliminate each others aboard one of our stations. On the bright side, those challengers are here to kill each other. On the downside, they don't care about how much collateral damage they cause, AND the victor may graduate to become a fully fledged Syndicate agent, enabling them to cause even more trouble. We ...
```
```suggestion
				<p>In a twisted parody of an interview process, the Syndicate's newest agents go through a so-called "Battle Royale" in which they attempt to eliminate one another aboard our stations. On the bright side, those challengers are here to kill each other. On the downside, they don't care about how much collateral damage they cause, AND the victor may graduate to become a fully fledged Syndicate agent, enabling them to cause even more trouble. We do not want that. Challengers often grimly display the corpse of their target on Newscasters. You may use this excess of hubris to try and figure out the identity of the victor so you may deal with them.
```

DeityLink replied:
A bit verbose but fitting, let's go

ShiftyRail commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+
+				<h2><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "synd-logo"))]' style='position: relative; top: 8;'><a id="Syndicate">The Syndicate</h2>
+
+                <p>The Syndicate are a collaboration of enemies to Nanotrasen. They are the most notorious threat to our space stations as they have made it their official goal to challenge Nanotrasen's monopoly on the sector. They are known to commit every possible crime from thievery, murder, sabotage, and the outright destruction of stations via nuclear devices. Have no qualms about killing th ...
```
```suggestion
                <p>The Syndicate is an ill-defined alliance of all enemies to Nanotrasen. They are the most notorious threat to our space stations as they have made it their official goal to challenge Nanotrasen's influence on the sector. No crime is below a Syndicate agent. Thievery, murder, sabotage, and the outright destruction of stations via nuclear devices are among their observed tactics. These enemies of the corporation are to be neutralised, detained, or in extreme cases, terminated on sight. Their red attire has been attributed to their scavenging of old soviet spacesuits from the fourth Cold War era.
```
Didn't like the open encouragement of validating on sight.

DeityLink replied:
I wouldn't call the Syndicate an "ill-defined" alliance. Since they are the threat that Nanotrasen has to deal with the most, we can expect them to know the most about them in turn. Ok for the rest.

ShiftyRail commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+                <p>The Syndicate are a collaboration of enemies to Nanotrasen. They are the most notorious threat to our space stations as they have made it their official goal to challenge Nanotrasen's monopoly on the sector. They are known to commit every possible crime from thievery, murder, sabotage, and the outright destruction of stations via nuclear devices. Have no qualms about killing th ...
+
+				<h3>Traitors</h3>
+				<p>One of the scariest traits of Syndicate agents is that most of them have infiltrated many of our space stations. Some manage to get hired to work at them, others get conscripted once there. They come equipped with an uplink filled with "telecrystals", or TC for short, usually hidden in their PDA, that lets them transport highly dangerous items aboard the station, usually in the goal of cau ...
```
```suggestion
				<p>Being unable to defeat NT in the open, Syndicate agents have opted to infiltrate many of our space stations and now live among us. The Syndicate's vast network of sleeper cells defies all investigation, and as such pre-emptive attempts at discovering infiltrators is discouraged by company policy (see Space Law and Standard Operating Procedure). Agents come equipped with an uplink filled with "telecrystals" (TCs), cleverly hidden in their PDAs. With such device, they are able transport highly dangerous items aboard the station, in their foolish attempts to cause wanton sabotage and destruction. Be aware of Syndicate contraband when apprehending suspects.
```

Less valid-hungry, reference to Space Law.

DeityLink replied:
The Syndicate is plenty capable of defeating NT in the open, it's just not their preferred tactic (as Nanotrasen probably has more resources).

ShiftyRail commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p>One of the scariest traits of Syndicate agents is that most of them have infiltrated many of our space stations. Some manage to get hired to work at them, others get conscripted once there. They come equipped with an uplink filled with "telecrystals", or TC for short, usually hidden in their PDA, that lets them transport highly dangerous items aboard the station, usually in the goal of cau ...
+
+				<h3>Nuclear Operatives</h3>
+				<p>A highly trained team of Syndicate agents, usually equipped with red rigsuits and 12mm SMGs. They are not only highly dangerous to fight but even to kill, as their brains are rigged with explosive implants that will try and take their aggressor's life with theirs, keep your distance as you fight them. Their goal is to acquire the station captain's nuclear authentication disk, so they can p ...
```
```suggestion
				<p>A highly trained team of Syndicate special agents, usually equipped with red rigsuits and 12mm SMGs. With blatant disrespect for the lives of their agents, the Syndicate has rigged their brains with explosive implants, in the goal of taking  their aggressor's life in their final breath. It is recommended to keep your distance as you fight them. The Operatives' goal is to acquire the station captain's nuclear authentication disk, so they can prime the nuclear fission device they carry in their ship, and blow the station up to smithereens. Confirmed sightings of operatives are grounds the crew with armory weapons as you will need all the firepower you can muster to fight them back. Sightings of operatives with cyborgs and mechas have been reported, against which it is recommended to use the ion rifles provided in your armory. Keep track of the disk's location at all times, and secure high priority locations such as the Armory, Medbay, Telecomms, and Engineering.
```

ShiftyRail commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p>The Syndicate trains its novice agents through a Battle Royale where they have to eliminate each others aboard one of our stations. On the bright side, those challengers are here to kill each other. On the downside, they don't care about how much collateral damage they cause, AND the victor may graduate to become a fully fledged Syndicate agent, enabling them to cause even more trouble. We ...
+
+				<h3>Elite Strike Team</h3>
+				<p>The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations. Sightings are rare, but they generally seem to field upgraded black rigsuits and extremely lethal ballistic weaponry. Their arrival is almost certain to spell doom for any station in a sector; do what you can to stop or delay them. 
```
```suggestion
				<p><b>Sensitive Information: Restricted Diffusion.</b>The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations. Sightings are rare, but they generally seem to field upgraded black rigsuits and extremely lethal ballistic weaponry. Their arrival is almost certain to spell doom for any station in a sector; do what you can to stop or delay them. 
```

Or just remove this bit all together.

ShiftyRail commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p>The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations. Sightings are rare, but they generally seem to field upgraded black rigsuits and extremely lethal ballistic weaponry. Their arrival is almost certain to spell doom for any station in a sector; do what you can to stop or delay them. 
+
+				<h2><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "vampire-logo"))]' style='position: relative; top: 8;'><a id="Vampire">The Vampire Lords</h2>
+				<p>We didn't want to believe it ourselves at first, but it has come to our attention that some employees have displayed what can only be described as "vampiric powers". We don't know for sure if those awakened here in space, or if they came from Earth and managed to fit among the migrants to our system, but they seem to exhibit a tendency to try and turn our space stations into their own pers ...
```
```suggestion
				<p>Rumours from less explored parts of the Galaxy talk about the so-called "masquerade" and its recent end . We don't know for sure if the vampires it refers to awakened out here in space, or if they are an old breed coming from Earth. In any case, they seem to exhibit a tendency to try and turn our space stations into their own personal fiefdoms - and blood banks. These undead lords have displayed a variety of powers, including summoning bats, hypnotization, and extensive healing capabilities.
```

ShiftyRail commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+
+				<h2><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "vampire-logo"))]' style='position: relative; top: 8;'><a id="Vampire">The Vampire Lords</h2>
+				<p>We didn't want to believe it ourselves at first, but it has come to our attention that some employees have displayed what can only be described as "vampiric powers". We don't know for sure if those awakened here in space, or if they came from Earth and managed to fit among the migrants to our system, but they seem to exhibit a tendency to try and turn our space stations into their own pers ...
+                <p>Because of those abilities, conventional ways of fighting prove often inefficient, fortunately, they appear to share the weakness of fairy tale vampires, namely an allergy to holy water, garlic, and mostly, they turn to dust in view of the sun, or in the presence of a deity. A great way to get rid of them permanently is thus to drag them in the station's chapel until their body ...
```
```suggestion
                <p>Due to the high cost of silver compared to lead and energy weaponry, Nanotrasen has declined to issue specific anti-vampire bullets. However, vampire lords have shown a surprising weakness to all that is Holy, regardless of creed and denomination. A great way to get rid of them permanently is thus to drag them in the station's chapel until their body turns to ashes. Holy Water and Harebells Incense also proves very efficient at harming them and hindering their abilities. Lastly, the Nullrods wielded by chaplains have shown efficiency at protecting them from those same powers.
```

ShiftyRail replied:
I would also remove all discussion of the specifics of how to kill the vamp. Just say they fear the holy items.

DeityLink replied:
I would argue that the specifics of how to kill the vamps are the most important information here.

ShiftyRail commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+                <p>Because of those abilities, conventional ways of fighting prove often inefficient, fortunately, they appear to share the weakness of fairy tale vampires, namely an allergy to holy water, garlic, and mostly, they turn to dust in view of the sun, or in the presence of a deity. A great way to get rid of them permanently is thus to drag them in the station's chapel until their body ...
+
+				<h3>Thralls</h3>
+				<p>Thralls are poor crewmembers who have been subjugated by a Vampire lord's powers into servitude. They will try to accomplish the will of their master and fight to defend them. Be ready to kill them, but if you manage to capture them alive, you may release them from their master's yoke by having them consume holy water. Keep making them drink it until you see their eyes clear up.
```
```suggestion
				<p>Thralls are poor crewmembers who have been subjugated by a Vampire lord's powers into servitude. They will try to accomplish the will of their master and fight to defend them. But Thralls are not beyond saving: blessed water has often been reported to be a potent antidote to a Vampire's influence.
```

DeityLink replied:
Alright

DeityLink commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+
+				<h2><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "synd-logo"))]' style='position: relative; top: 8;'><a id="Syndicate">The Syndicate</h2>
+
+                <p>The Syndicate is an ill-defined alliance of all enemies to Nanotrasen. They are the most notorious threat to our space stations as they have made it their official goal to challenge Nanotrasen's influence on the sector. No crime is below a Syndicate agent. Thievery, murder, sabotage, and the outright destruction of stations via nuclear devices are among their observed tactics.  ...
```
```suggestion
                <p>The Syndicate is a collaboration of enemy corporations to Nanotrasen. They are the most notorious threat to our space stations as they have made it their official goal to challenge Nanotrasen's influence on the sector. No crime is below a Syndicate agent. Thievery, murder, sabotage, and the outright destruction of stations via nuclear devices are among their observed tactics. These enemies of the corporation are to be neutralised, detained, or in extreme cases, terminated on sight. Their red attire has been attributed to their scavenging of old soviet spacesuits from the fourth Cold War era.
```

DeityLink commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+                <p>The Syndicate is a collaboration of enemy corporations to Nanotrasen. They are the most notorious threat to our space stations as they have made it their official goal to challenge Nanotrasen's influence on the sector. No crime is below a Syndicate agent. Thievery, murder, sabotage, and the outright destruction of stations via nuclear devices are among their observed tactics. T ...
+
+				<h3>Traitors</h3>
+				<p>Being unable to defeat NT in the open, Syndicate agents have opted to infiltrate many of our space stations and now live among us. The Syndicate's vast network of sleeper cells defies all investigation, and as such pre-emptive attempts at discovering infiltrators is discouraged by company policy (see Space Law and Standard Operating Procedure). Agents come equipped with an uplink filled wi ...
```
```suggestion
				<p>As a matter of fact, Syndicate agents have infiltrated many of our space stations and now live among us. The Syndicate's vast network of sleeper cells defies all investigation, and as such pre-emptive attempts at discovering infiltrators is discouraged by company policy (see Space Law and Standard Operating Procedure). Agents come equipped with an uplink filled with "telecrystals" (TCs), cleverly hidden in their PDAs. With such device, they are able transport highly dangerous items aboard the station, in their foolish attempts to cause wanton sabotage and destruction. Be aware of Syndicate contraband when apprehending suspects.
```

DeityLink commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p>As a matter of fact, Syndicate agents have infiltrated many of our space stations and now live among us. The Syndicate's vast network of sleeper cells defies all investigation, and as such pre-emptive attempts at discovering infiltrators is discouraged by company policy (see Space Law and Standard Operating Procedure). Agents come equipped with an uplink filled with "telecrystals" (TCs), c ...
+
+				<h3>Nuclear Operatives</h3>
+				<p>A highly trained team of Syndicate special agents, usually equipped with red rigsuits and 12mm SMGs. With blatant disrespect for the lives of their agents, the Syndicate has rigged their brains with explosive implants, in the goal of taking  their aggressor's life in their final breath. It is recommended to keep your distance as you fight them. The Operatives' goal is to acquire the statio ...
```
```suggestion
				<p>A highly trained team of Syndicate special agents, usually equipped with red rigsuits and 12mm SMGs. With blatant disrespect for the lives of their agents, the Syndicate has rigged their brains with explosive implants, in the goal of taking  their aggressor's life in their final breath. It is recommended to keep your distance as you fight them. The Operatives' goal is to acquire the station captain's nuclear authentication disk, so they can prime the nuclear fission device they carry in their ship, and blow the station up to smithereens. Confirmed sightings of operatives are grounds to provide the crew with armory weapons as you will need all the firepower you can muster to fight them back. Sightings of operatives with cyborgs and mechas have been reported, against which it is recommended to use the ion rifles provided in your armory. Keep track of the disk's location at all times, and secure high priority locations such as the Armory, Medbay, Telecomms, and Engineering.
```

ShiftyRail commented:
>he didn't keep the Among Us

😔

DeityLink commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p>In a twisted parody of an interview process, the Syndicate's newest agents go through a so-called "Battle Royale" in which they attempt to eliminate one another aboard our stations. On the bright side, those challengers are here to kill each other. On the downside, they don't care about how much collateral damage they cause, AND the victor may graduate to become a fully fledged Syndicate a ...
+
+				<h3>Elite Strike Team</h3>
+				<p><b>Sensitive Information: Restricted Diffusion.</b>The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations. Sightings are rare, but they generally seem to field upgraded black rigsuits and extremely lethal ballistic weaponry. Their arrival is almost certain to spell doom for any station in a sector; do what you can to sto ...
```
```suggestion
				<p><b>Sensitive Information: Restricted Diffusion.</b> The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations. Sightings are rare, but they generally seem to field upgraded black rigsuits and extremely lethal ballistic weaponry. Their arrival is almost certain to spell doom for any station in a sector; do what you can to stop or delay them. 
```

DeityLink commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p><b>Sensitive Information: Restricted Diffusion.</b> The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations. Sightings are rare, but they generally seem to field upgraded black rigsuits and extremely lethal ballistic weaponry. Their arrival is almost certain to spell doom for any station in a sector; do what you can to st ...
+
+				<h2><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "vampire-logo"))]' style='position: relative; top: 8;'><a id="Vampire">The Vampire Lords</h2>
+				<p>Rumours from less explored parts of the Galaxy talk about the so-called "masquerade" and its recent end . We don't know for sure if the vampires it refers to awakened out here in space, or if they are an old breed coming from Earth. In any case, they seem to exhibit a tendency to try and turn our space stations into their own personal fiefdoms - and blood banks. These undead lords have dis ...
```
```suggestion
				<p>Rumours from less explored parts of the Galaxy talk about the so-called "masquerade" and its recent end. We don't know for sure if the vampires it refers to awakened out here in space, or if they are an old breed coming from Earth. In any case, they seem to exhibit a tendency to try and turn our space stations into their own personal fiefdoms - and blood banks. These undead lords have displayed a variety of powers, including summoning bats, hypnotization, and extensive healing capabilities.
```

DeityLink commented:
@ShiftyRail 
> > he didn't keep the Among Us
> 
> 😔
I totally kept it!

Eneocho commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+				<p><b>Sensitive Information: Restricted Diffusion.</b> The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations. Sightings are rare, but they generally seem to field upgraded black rigsuits and extremely lethal ballistic weaponry. Their arrival is almost certain to spell doom for any station in a sector; do what you can to st ...
+
+				<h2><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "vampire-logo"))]' style='position: relative; top: 8;'><a id="Vampire">The Vampire Lords</h2>
+				<p>Rumours from less explored parts of the Galaxy talk about the so-called "masquerade" and its recent end. We don't know for sure if the vampires it refers to awakened out here in space, or if they are an old breed coming from Earth. In any case, they seem to exhibit a tendency to try and turn our space stations into their own personal fiefdoms - and blood banks. These undead lords have disp ...
```
I would change the "hypnotization" with "hypnosis".

DeityLink replied:
```suggestion
				<p>Rumours from less explored parts of the Galaxy talk about the so-called "masquerade" and its recent end. We don't know for sure if the vampires it refers to awakened out here in space, or if they are an old breed coming from Earth. In any case, they seem to exhibit a tendency to try and turn our space stations into their own personal fiefdoms - and blood banks. These undead lords have displayed a variety of powers, including summoning bats, hypnosis, and extensive healing capabilities even from a seemingly dead state.
```

Theophrastus1 commented:
> A lot of sentences are really weirdly phrased, but this is a good change. Also, it's "Battle Royale", not "Battle Royal", on the challengers' tab.

fucked up that a frenchman gets a french loan word wrong

ManopolisJones commented:
>Feral Catbeasts
>Not JUST Catbeasts
Lets not pretend like these fucking animals are tolerable in any form, let alone feral. Also, using the term 'bad mojo' in place of 'bad luck' really laments the overall writing in this, where the whole thing has good advice on antags weaknesses and how to deal with them (such as holy water on thralls, how wizard clothes are tied to certain powers, ect.) but the actual writing and fluff is lackluster enough where it feels like it could have used some more workshopping before being put forth.

DeityLink commented:
> Lets not pretend like these fucking animals are tolerable in any form, let alone feral. Also, using the term 'bad mojo' in place of 'bad luck' really laments the overall writing in this, where the whole thing has good advice on antags weaknesses and how to deal with them (such as holy water on thralls, how wizard clothes are tied to certain powers, ect.) but the actual writing and fluff is lackluster enough where it feels like it could have used some more workshopping before being put forth.

It was put forth explicitely to benefit from suggestions from the community. I'm not a narrative designer ok? I spent 4 hours writing most of it and getting the html style in a place I was personally happy with. The writing sits well enough with me and the rest is down to individual preferences. Also If catbeasts were that intolerable sec would just raid every custom bar that prays admins for holographic dancing tajarans. I try to remain vague enough to allow a certain degree of headcanon to each of  our players.

PrimeDSS13 commented:
I'm not sure I like a canon "book of antags", really. Just feels out of place.

Theophrastus1 commented:
I'm also not a fan of the big book of antags, considering it seems a bit, i dunno. It's hard for me to put into words. Especially for things like minor roles.

ErrorsWindows commented:
good man, might consider adding some pictures or diagrams, maybe get it looking like less of a text wall

DeityLink commented:
Some pictures would be nice, I could reduce the "wall of text"ness by having each category fit on its own page, the same way cult tomes do.

ErrorsWindows commented:
list any procedure for what to do or how to stop them or how to nonlethally detain them or if they can even be detained? or just a quick overview for IFF purposes and let the more experienced characters know what to actually do

ManopolisJones commented:
> It was put forth explicitly to benefit from suggestions from the community. I'm a gameplay programmer/designer not a narrative designer nor a native english speaker....

Forgive me for the late response (and the length of this upcoming one), however I feel something like this does indeed require a certain sense of narrative design as you put it. This is reasoning will be difficult, but the way I see it, when something like this book comes forth, our sense of subjectivity on these antagonists, or 'headcannon' as you put it, begins to diminish, more so in the minds of new players who might take such a book as word of law on the subject. The reason I feel this way, is, really this isn't a library book where you pull it out of the machine and you go in knowing somebody made it as their interpretation of such (even if it ostensibly is), for it is put in, in an official capacity, much like that of a book with a wiki guide in a department, or something like our book on space law.

Going further beyond that, and more into the point of narrative design, I would not feel so strongly about this book, if I felt it was well written. Going through the PDF, you see a lot of what I could only describe as "lax verbiage", where in the document is overly casual on something that should be more straight laced and matter of fact. Look at lines like "You'll have to bear with us on that one, but as it turns out..." on the wizard federation, or, "We didn't want to believe it ourselves at first, but it has come to our attention..." from the vampire lords one, these are something someone who is talking to you casually would be saying, not a matter of fact document on the subject of our enemies. Now I can identify not everything in this document can stay purely in the realm of fact, and that opinion is inevitable as we reach the fringes of accepted knowledge (this being mostly in the area of minor antagonists, such as time agents or soul ramblers) but it is at this point where good and in the case of our server, humorous writing is really required to keep the reader immersed in the game they're in.

I believe if parts of the book are unable to stay as dry and factual as humanly possible, that is to leave most of the lore bits to the imagination of the reader, then it must be funny and well written enough to where the reader won't care if it goes counter to their beliefs on the subject (after all headcannon exists to go counter to whatever's written in stone). With all this being said, and I feel strongly that I might be able to say more on the topic, this entire PDF would be better suited as a wiki article "Antag Rundown" then an actual book you can pick up and read next to a copy of space law in game. Because frankly speaking, for all the reasons I gave, I do not think this has the level writing in it to stack up as an official piece of NT lore on our server (if that's even worth a damn to anybody).

DeityLink commented:
> after all headcannon exists to go counter to whatever's written in stone

Not necessarily, some use the term to tell about their idea about elements that are omitted from the source material.

> I do not think this has the level writing in it to stack up as an official piece of NT lore on our server (if that's even worth a damn to anybody).

Before I ask if anyone else here wants to give a shot at giving it a narrative pass for free, do we have currently in the game any "official piece of NT lore" that I could use as a reference?

ErrorsWindows commented:
damn and here i spent all this time getting welcomed into bay only for the lore-writer book club to come here now

DeityLink commented:
4th iteration featuring a more "professional" entry for Wizards and Grue.

DeityLink commented:
5th iteration, rewrote the intro to vampires

Inorien commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,165 @@
+
+				<h1><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "nano-logo"))]' style='position: relative; top: 6;'>Enemies of Nanotrasen: A Quick Overview<img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "nano-logo"))]' style='position: relative; top: 6;'></h1>
+                <h2>Foreword</h2>
+				<p>A Mega-Corporation the size of Nanotrasen doesn't carve itself a piece of the intergalactic cake without making itself a few enemies along the way. Identifying agents of those enemy factions and thwarting their plans falls into the job description of our Security forces, but every Nanotrasen employee is encouraged to report any suspicious activities, and should the situation call for it, t ...
```
A mega-corporation as successful as Nanotrasen does not cut itself a slice of the intergalactic cake without attracting the attention of undesirable forces. Identifying and neutralising threats to the station and other company assets is the prime responsibility of Security forces, however every employee is expected to do their part in protecting the station to which they have been assigned. This guide serves as a primer on various dangers you may encounter in the course of your duties.

^ is this splitting hairs? I feel like using the word "enemy" implies there's a war or something, but depending on how propaganda-heavy we imagine NT to be, I could see them trying to discredit the Syndicate and such by just referring to them as "terrorists" or something similar

cloutcode commented:
Not sure why this book is a problem when every single textbook in the game is just a direct link to the wiki

Eneocho commented:
Cause this book is NOT a link to the wiki (I.e: an external source), it's a direct in-game lore/canon book.

DeityLink commented:
It's not canon, not any more than the PDA Events that nobody reads or any other piece of text written by some bloke coder.

Hinaichigo commented:
It does sort of imply that these things occur regularly enough to be considered "known" which for the minor antags especially seems a bit strange.  Like should soul ramblers be documented by official Nanotrasen guidelines? I was on the fence about this for a bit, but it does collapse the possible space of headcanons. Like from an IC perspective, instead of something being truly out of the ordinary and unknown to the player characters, it becomes a "known factor" that the players and NT have readily available knowledge about and have to deal with occasionally. Reduces the RP-space by a little bit. Plus it might give players who don't know about all the antags or don't read the wiki, or are new, an idea that these are all of the antags available acting like a "spoiler" to an extent. And having it in sec sorta feeds even more into gradual slip down the slope of the security = protagonists concept which I don't really like.

DeityLink commented:
ss13 "lore" is full of plotholes and players routinely ignore whatever doesn't suit their current gimmick. This manual isn't gonna change that @Hinaichigo I assure you, and admins aren't gonna enforce it either. It does give something players can RP onto, but only if they want to.

More importantly, I'm suprised the IC aspect is what players get so hung up about, the main purpose of this manual isn't to provide lore, but to give concrete knowledge on how to recognize and deal with the various antags, so if some paragraph in the manual goes too far in the lore department feel free to make a suggestion and I will make another revision. I've already applied suggestions from Shifty, Sonix, Bathos, and applied some of Manopolis' advices to make the Wizards/Vampire/Grue paragraph more "professional".

> And having it in sec sorta feeds even more into gradual slip down the slope of the security = protagonists concept which I don't really like.

It's in the library as well. Also dealing with antagonists IS kinda expected of sec, poes that make sec the protagonists? I wouldn't be so sure. 

> If anything, each one could be its own book with its own flavor. Like a book about vampires would be a tome written by a vampire hunter long ago. A book about the blob could be like some corrupted data slate jettisoned out of a doomed station. But then should all of these just be lying in the library? Maybe they could be something the trader brings, or something you can find exploring ruins or derelicts in space.

The problem here is that I want the practical informations in this book to be readily available to players.

Lore books that you can buy from traders or find in space would be neat, but unrelated to what I'm trying to do here. This is only "IC" in the "trying not to break immersion" way.

Hinaichigo commented:
> The problem here is that I want the practical informations in this book to be readily available to players.

The practical information is already readily available to _players_ via the wiki and other resources. The distinction is whether or not it should be readily available to _characters_. Practical job manuals laying around the various workstations make sense because they represent and describe the "ordinary" world of a functional station that we find ourselves in at roundstart. However, antagonists essentially represent a disturbance or anomaly with regard to that "quotidian" state; something going wrong. To canonically bring the antagonists into the fold of "a typical shift on the station" by having them described in a litanic roster of sorts, is to run counter to that essence.

DeityLink commented:
> The practical information is already readily available to players via the wiki and other resources. 

Don't act dumb, I mean OFF the wiki, and in-game. Most players on vg have made it clear for years they don't want to "pretend" not knowing how to fire a gun, or how to do a job that isn't their assigned job if the situation calls for it. Those who do are the exception. 

> To canonically bring the antagonists into the fold of "a typical shift on the station" by having them described in a litanic roster of sorts, is to run counter to that essence.

Say hello to the "System Federation Status Summary" that literally warns the station of the presence of antags _every single round_. I don't recall THAT being much of an issue back then.

![image](https://github.com/vgstation-coders/vgstation13/assets/7573912/6d0bbe33-846f-42e3-a874-cc6e79061c06)

DeityLink commented:
Oh this one is good too
![image](https://github.com/vgstation-coders/vgstation13/assets/7573912/ef0b0cb9-84e7-4243-a105-e65634bb129e)

DeityLink commented:
I'll do a global pass tonight to improve the style following advice I'm currently receiving from various players.

DeityLink commented:
[SixthRevision.pdf](https://github.com/vgstation-coders/vgstation13/files/12100755/SixthRevision.pdf)

6th revision is up, this time I did a pass on most sections to help them sound more professional and technical and I think it's in a good place now. Thank you to all of those who have provided feedback.

pedr69 commented:
>blob
>The nuclear authentication code is provided to everyone

huh? unlinked borgs don't get the code, neither does the crew
only AIs do

DeityLink commented:
ah, I double-checked the code and you're partially right. It's not included in the global announcement, however it is also printed to communications consoles. I'll make some corrections.

DeityLink commented:
[seventhrevision.pdf](https://github.com/vgstation-coders/vgstation13/files/12177044/seventhrevision.pdf)

Fixed the above inaccuracy. Also added a mention of Directive 7-10 specifying that the station is under quarantine during Blob.
Also added some Vampire tweaks regarding their ability and that drinking blood makes them stronger (how did I forget that part?)

DeityLink commented on code/game/objects/items/weapons/manuals.dm:
```diff
@@ -499,6 +499,173 @@
+			  <h3>Conversions and Deconversions</h3>
+			  <p>During our early investigation of the cult, we've observed that their numbers aboard a station often grew larger over time. We have deduced that they have the ability to forcefully convert crew members into service to their god. Thankfully our patented loyalty implants appear to guarantee that our crew may resist conversion attempts, so you should run loyalty implantation campaigns if a c ...
+              <p>Thankfully, the conversion process is reversible. A committee of chaplains has reached to us to share a couple rituals that anyone can perform to exorcize the yoke of Nar-Sie from a cultist: Give the target a sip of holy water, then apply a Bible on their head. The ritual may or may not succeed, as the cultists can resist it, so be prepared for the occult consequences of a failed ...
+              <p>The second ritual allows us to know how large is the current cult presence in the sector: by taking blood from a living cultist and mixing it with some holy salts, this causes a reaction manifesting red and orange flames. Each red flame corresponds to a living cultist or construct aboard the station, while each orange flame corresponds to either one of them off station, or to a s ...
```
```suggestion
              <p>The second ritual allows us to know how large is the current cult presence in the sector: by dropping 5 units of holy salts into a sample of blood from a living cultist, a reaction manifesting red and orange flames occurs. Each red flame corresponds to a living cultist or construct aboard the station, while each orange flame corresponds to either one of them off station, or to a shade. A cultist can be kept alive and rendered mostly harmless by injecting them with a holy implant.
```

DeityLink commented:
Improved Foreword and Syndie paragraphs courtesy of Optimism

[eighthrevision.pdf](https://github.com/vgstation-coders/vgstation13/files/12268340/eighthrevision.pdf)

Optimism333 commented:
I'm doing some cooperative proofreading and editing with Deity to help improve the guide.

I will try to get through every section before a merge if possible. If not, I will continue after the merge.
